lib/editor/actions/editor.js

```diff
@@ -31,6 +31,11 @@ import type {
 
 import {clearGtfsContent, saveActiveGtfsEntity, setActiveGtfsEntity} from './active'
 
+const LOCK_API_PATH = '/api/editor/secure/lock'
+// Identifies what is being locked.
+export const GTFS_EDITOR_LOCK = 'gtfs-editor'
+export const GTFS_PLUS_EDITOR_LOCK = 'gtfs-lock-editor'
```

Comment:
Why is the second string not `gtfs-plus-editor`?

Reply:
Yikes - fixed in https://github.com/ibi-group/datatools-ui/pull/852/commits/da9d219113c3b89f02471f3f9ab1becc0b1e2fa7.